.github/workflows/bumpversion.yml

```diff
@@ -0,0 +1,36 @@
+name: Bump_Version
+
+on:
+  repository_dispatch:
+    types: [kolibri-explore-plugin]
+
+jobs:
+  bump-version:
+    runs-on: windows-latest
+
+    steps:
+      - name: Set up Git environment
+        run: |
+          git config --global user.name ${{ github.actor }}
+          git config --global user.email ${{ github.actor }}@github.com
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # When there are multiple matched tags at the same commit, "git describe"
+      # only gets the first matched tag, not the latest one. That leads generate
+      # conflicted tag next time, due to the same tag name. So, append a dummy
+      # commit here.
+      - name: Append a new commit for new kolibri-explore-plugin
+        if: ${{ github.event.action == 'kolibri-explore-plugin' }}
```

Comment:
Is `github.event.action` really going to be `kolibri-explore-plugin`? It seems like you want `github.event.event_type` per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch.

Reply:
I used `github.event.event_type` at first try, but it showed nothing.  Then, did some study and found it is kind of `github.event.action`.
Here is my repo's example:
* Code: https://github.com/starnight/build-image/blob/v0.0.15/.github/workflows/bumpversion.yml#L27
* Debug run: https://github.com/starnight/build-image/actions/runs/5208911111/jobs/9398076874#step:4:18